pkgs/characters/lib/src/characters.dart

```diff
@@ -19,7 +19,7 @@ import "characters_impl.dart";
 /// which allows iterating the independent characters in both directions,
 /// but which also provides ways to select other ranges of characters
 /// in different ways.
-abstract class Characters implements Iterable<String> {
+abstract interface class Characters implements Iterable<String> {
```

Comment:
Can we move this to a separate commit? Or if not, please include a note about this change in the commit message. I think we should also include it in the changelog.

Reply:
Will revert for now.
It's not like it's *useful* to extend an empty class, but people do ... things.